src/UI/Component/ViewControl/Factory.php

```diff
@@ -64,4 +64,38 @@ public function mode($labelled_actions, $aria_label);
 	 */
 	public function section(Button $previous_action, Button $button, Button $next_action);
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *      The sortation view control enables users to change the order in which
+	 *      some data is presented.
+	 *      The common table-sorting mechanism, e.g., in which a column header is clicked,
+	 *      requires the column to be visible. Visibility and sorting can be decoupled
+	 *      by a control next to the actual table.
+	 *      This concept is used by many pages, notably in shops like e.g. amazon, and can
+	 *      be applied to not only tables but all kinds of data-listings.
+	 *   composition: >
+	 *      Sortation uses a Dropdown to display a collection of shy-buttons.
+	 *   effect: >
+	 *      A click on an option will change the ordering of the associated data-list
+	 *      by calling the current page with the parameter "sortation" and its
+	 *      value according to the selected option.
+	 *      The label displayed in the dropdown will be set to the selected sorting.
+	 *
+	 * rules:
+	 *   usage:
+	 *      1: A Sortation MUST NOT be used standalone.
+	 *      2: Sortations MUST BE visually close to the list or table their operation will have effect upon.
+	 *      3: There SHOULD NOT be more than one Sortation per page.
```

Comment:
Recently we had a page vs view vs screen discussion in an other group. To you really mean "Page" here (all that ILIAS outputs)? I think you could have more than one sorts in a page (e.g. for something in a sidebar and the center stage), but probably only one for a certain view on something (e.g. an Appointment Listing).